documentation/docs/client/pinned_messages.md

```diff
@@ -0,0 +1,80 @@
+---
+id: clientPinnedMessages
+title: Pinned Messages
+sidebar_position: 22
+---
+
+Pinned messages allow users to highlight important messages, make announcements, or temporarily promote content. Pinning a message is, by default, restricted to certain user roles, but this is flexible. Each channel can have multiple pinned messages and these can be created or updated with or without an expiration.
+
+## Pinning a Message
+
+An existing message can be updated to be pinned or unpinned by using the `channel.pinMessage` and `channel.unpinMessage` methods. Or a new message can be pinned when it is sent by setting the pinned and `pin_expires` fields when using `channel.sendMessage`.
```

Comment:
```suggestion
An existing message can be updated to be pinned or unpinned by using the `channel.pinMessage` and `channel.unpinMessage` methods. 
A new message can be pinned when it is sent by setting the `pinned` and `pin_expires` fields.
```

Reply:
Thanks, also update `channel ` to `channelClient` and `pin_expires` to `pinExpires`.